src/request.ts

```diff
@@ -10,6 +10,10 @@ import { gzipSync, strToU8 } from 'fflate'
 // eslint-disable-next-line compat/compat
 export const SUPPORTS_REQUEST = !!XMLHttpRequest || !!fetch
 
+const CT_PLAIN = 'text/plain'
+const CT_JSON = 'application/json'
+const CT_FORM = 'application/x-www-form-urlencoded'
```

Comment:
very nitpicky that the bundler can minify `CT_BLAH` to Xq or whatever so we could have `CONTENT_TYPE_BLAH` for readdability

But it can't minify `application` so we _might_ save a few bytes by having `APPLICATION + '/json'` and `APPLICATION + '/x-www-etc`

(frustratingly the bundler converts `${APPLICATION}/json` to `"".concat(APPLICATION, "/json")` so using template strings doesn't save much bundle

I'm not super keen on bundle code golf so consider this the nittiest of picks